src/VCR/Util/CurlHelper.php

```diff
@@ -175,24 +175,39 @@ public static function setCurlOptionOnRequest(Request $request, $option, $value,
             case CURLOPT_HEADER:
             case CURLOPT_WRITEFUNCTION:
             case CURLOPT_HEADERFUNCTION:
+            case CURLOPT_UPLOAD:
```

Comment:
As far as I see [`CURLOPT_UPLOAD`](http://curl.haxx.se/libcurl/c/CURLOPT_UPLOAD.html) should be used when uploading a file in conjunction with a read function. So it should be still possible to upload files using the `@` notation when we ignore the option.